### PR TITLE
LPS-172892 Migrate Object's Action tab to React

### PR DESCRIPTION
**Pull request by igor-franca**
Related [Ticket](https://issues.liferay.com/browse/LPS-172892)

### CODE REVIEW

**Comment by igor-franca** [comment]
[Depends on](https://github.com/liferay-objects/liferay-portal/pull/2145)

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
Investigating the errors

**Comment by igor-franca** [comment]
ci:test:object

**Comment by naraandrade** [comment]
Wait for https://github.com/liferay-objects/liferay-portal/pull/2145

**Comment by naraandrade** [comment]
Just started reviewing :)

**Review by naraandrade — CHANGES_REQUESTED** [review]
@igor-franca There's an intermittent issue when we first open the Action tab. The sidebar seems to open and close real quick. Could you take a look at it? Thanks :)

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/Actions.tsx [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+};
+interface ItemData {
+	active: boolean;
+	defaultObjectAction: boolean;
+	id: number;
+	label: LocalizedValue<string>;
+	status: Status;
+}
+
+export default function Action({
```
Comment (naraandrade) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/Actions.tsx:45]: @igor-franca Please, change the name of the component to "Actions".

**Review by naraandrade — CHANGES_REQUESTED** [review]

**Review by naraandrade — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:test:object

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:forward

**Comment by naraandrade** [comment]
Failures not related. I'm going to send this PR directly to BChan.

**Comment by naraandrade** [comment]
Just started reviewing :)

**Comment by naraandrade** [comment]
Resent here: https://github.com/brianchandotcom/liferay-portal/pull/136242

**Pull request closed without merging**